astroquery/utils/tap/core.py

```diff
@@ -173,7 +173,7 @@ def load_table(self, table, verbose=False):
         """
         if table is None:
             raise ValueError("Table name is required")
-        print("Retrieving table '{}'".format(table))
+        log.info("Retrieving table '{}'".format(table))

```

Comment:
Given that there is already a `verbose` kwarg here, instead of the log.info, please use the verbose and the print (parts of this file already do that)